scss/_patterns_split-list.scss

```diff
@@ -0,0 +1,60 @@
+@import 'settings';
+
+@mixin vf-p-split-list {
+  * {
+    // border: 1px solid red;
+  }
+
+  .p-split-list-classes {
+    @extend .u-fixed-width;
+
+    .p-split-list__main-hr {
+      @extend .u-hide--small;
+    }
+
+    .p-split-list__main-row {
+      @extend .row;
+    }
+
+    .p-split-list__main-col {
+      &:first-child {
+        @extend .col;
```

Comment:
Not sure what it does? `.col` is used in `row--x-x` shorthands, if we mean full width column we should make it more explicit with `.col-12`.